packages/smart-contracts/test/contracts/BatchConversionPayments.test.ts

```diff
@@ -829,37 +803,37 @@ describe('contract: BatchConversionPayments', async () => {
 
       const [expectedFromFAUBalanceDiff, expectedToFAUBalanceDiff, expectedFeeFAUBalanceDiff] =
         getExpectedERC20Balances(100000, 100, 1);
-      await calculDiffAndCheckERC20Balances(
+      await expectERC20BalanceDiffs(
         'FAU',
-        fromOldFAUBalance,
-        toOldFAUBalance,
-        feeOldFAUBalance,
+        initialFromFAUBalance,
+        initialToFAUBalance,
+        initialFeeFAUBalance,
         expectedFromFAUBalanceDiff,
         expectedToFAUBalanceDiff,
         expectedFeeFAUBalanceDiff,
       );
     });
 
-    it('batchEthPayments make 1 ETH payment without conversion', async function () {
+    it('make 1 ETH payment without conversion', async function () {
       // get Eth balances
-      const beforeETHBalanceTo = await provider.getBalance(to);
-      const beforeETHBalanceFee = await provider.getBalance(feeAddress);
-      const beforeETHBalanceFrom = await provider.getBalance(await fromSigner.getAddress());
+      const initialToETHBalance = await provider.getBalance(to);
+      const initialFeeETHBalance = await provider.getBalance(feeAddress);
+      const initialFromETHBalance = await provider.getBalance(await fromSigner.getAddress());
       tx = await batchConversionProxy.batchEthPayments(
         [to],
         ['1000'],
         [referenceExample],
         ['1'],
         feeAddress,
-        { value: 1000 + 1 + 42 },
+        { value: 1000 + 1 + 42 }, // +42 in excess
```

Comment:
```suggestion
        { value: 1000 + 1 + 42 }, // +42 in excess should not impact the final amount exchanged
```

Reply:
It is necessary because of the batch fees ( I could have 1001 * 10% -> about  +11 )